src/l500/l500-options.h

```diff
@@ -30,31 +30,37 @@ namespace librealsense
     };
 
     class l500_options;
+    class l500_hw_options;
 
     typedef uvc_xu_option< int > super;
```

Comment:
`super` should only be defined as the private of a class, never outside

Reply:
done